src/LinkLight.js

```diff
@@ -9,7 +9,6 @@ import styled from 'react-emotion'
 
 const LinkLight = styled(Link)`
   color: ${props => props.theme.colors.blue[3]} !important;
-  ${fontSize}
 `
 
 LinkLight.defaultProps = {
```

Comment:
I think I'm not even using these defaultProps if you feel like removing them 🤷‍♀️